app/scripts/views/form.js

```diff
@@ -214,7 +214,7 @@ function (_, $, p, Validate, AuthErrors, BaseView, Tooltip,
         return false;
       }
 
-      var inputEls = this.$('input');
+      var inputEls = this.$('input').not('#coppa input');
```

Comment:
see https://github.com/mozilla/fxa-content-server/pull/2977#discussion_r37354501

We want to validate those fields, except with the nested views it is not doing the right thing. 
It was a `select` for the dropdown so this selector did not catch those.